WordPress/Classes/ViewRelated/QR Login/Helpers/QRLoginCameraSession.swift

```diff
@@ -24,7 +24,9 @@ class QRLoginCameraSession: NSObject, QRCodeScanningSession {
     }
 
     func start() {
-        session?.startRunning()
+        DispatchQueue.global(qos: .background).async { [weak self] in
```

Comment:
I think we should use a serial queue to start and stop the camera session. See [this Apple sample project](https://developer.apple.com/documentation/avfoundation/capture_setup/avcam_building_a_camera_app).